mocks/mocks.py

```diff
@@ -243,63 +243,63 @@ def scale_light_by_distance(particle_positions,
     return particle_positions.value, particle_values
 
 
-def prepare_mocks(config):
-    """
-    Creates a dictionary containing configuration data and a numpy.array of
-    the simulation data.
-
-    Parameters
-    ----------
-    config : dict
-        A dictionary of configuration items.
-
-    Returns
-    -------
-    mock_image_input: dict
-    galaxy_sim_data_raw : numpy.array
-        Prepared information for creating mock images and the simulation data.
-
-    Deleted Parameters
-    ------------------
-    config_location : str, optional
-        The name (location) of the yaml file that contains initial
-        information, default `config_example.yaml`.
-    """
-    # Creating the cosmology
-    cosmo = create_cosmology(config)
-
-    z = compute_redshift(config['galaxy_distance'],
-                         cosmo)
-
-    # Computing the pixel scale.
-    config['pixel_scale'] = compute_pixel_scale(z,
-                                                cosmo,
-                                                config['sim_pc_\
-pixel'])
-    # Reading the data.
-    with fits.open(config['data_path']) as sim_fits:
-        galaxy_sim_data_raw = sim_fits[0].data
-    # Computing the total mass of the galaxy:
-    galaxy_mass = compute_total_mass(galaxy_sim_data_raw,
-                                     config['particle_baryonic_mass_sim'],
-                                     mass_factor=1e5,
-                                     H=config['hubble_constant'])
-
-    band = config['imager_filter']
-    # Mass to light ratio for demanded band (filter).
-    mass_to_light = config['mass_to_light_ratio'][band]
-    # Absolute magnitude of the Sun for the demanded band.
-    abs_mag_sun = config['abs_mag_sun'][band]
-    # Total apparent ABmag of the simulated galaxy in the demanded band.
-    total_apparent_ABmag_sim = compute_apparent_ABmag(z,
-                                                      galaxy_mass,
-                                                      mass_to_light,
-                                                      abs_mag_sun,
-                                                      cosmo)
-
-    config['total_mag'] = total_apparent_ABmag_sim
-
-    return config, galaxy_sim_data_raw
+# def prepare_mocks(config):
```

Comment:
With respect to the jupyter outflow, which is provided in #71 , there will be no need for this function anymore. But I will remove it and its test when this PR is approved.